tests/test_playwright.py

```diff
@@ -380,3 +382,18 @@ def test_device_emulation(page, device):
     )
     result = testdir.runpytest("--device", "iPhone 11 Pro")
     result.assert_outcomes(passed=1)
+
+
+def test_trace_dir(testdir: pytest.Testdir, tmpdir: Any) -> None:
```

Comment:
`tmpdir` should be annotated with `Path` from `pathlib`

Reply:
Ah maybe I should use pytests' `tmp_path` rather than `tmpdir`. https://docs.pytest.org/en/6.2.x/tmpdir.html#the-tmp-path-fixture

`tmp_path` does return a `path_lib.Path` object. But `tmpdir` https://docs.pytest.org/en/6.2.x/tmpdir.html#the-tmpdir-fixture returns "a py.path.local object". I wasn't what type that was. Ah it's from here https://py.readthedocs.io/en/latest/path.html

Yeah I could probably switch to using `tmp_path` and then `Path` would be the right annotation. I think tmpdir is maybe an older fixture.